### DISCUSSION TITLE
Chrome & Firefox extension toolbar icon does not show match count

### DISCUSSION THREAD

**Discussion by truthsword** [dani-garcia/vaultwarden#3031] [Q&A]
Using latest version:
![](https://i.imgur.com/AqksKGi.png)

When landing on a site (ex. github.com) for which login credentials exist, the browser extension icon no longer shows count of available logins.
![](https://i.imgur.com/Biz3h36.png)
Please advise.

**Comment by tessus** [comment]
https://github.com/bitwarden/clients/issues/4260

**Reply by truthsword** [reply]
Badge counts are back today on 2 PCs, with Chrome and Firefox.

**Reply by tessus** [reply]
Nope, I don't think that's possible (they haven't fixed the issue yet) unless you downgraded to 2011.10.0. Or you are only using one Firefox/Chrome window. If you have more than 1 window, the badge does not work on the other windows.

**Reply by truthsword** [reply]
You are correct. I didn't realize window count mattered, and I did think it odd that my extension version had not changed.

**Comment by tessus** [comment]
Version 2023.1.0 of the browser add-on fixes the issue.
